aea/skills/base.py

```diff
@@ -179,6 +179,14 @@ def __init__(self, **kwargs):
         """
         self._context = kwargs.pop("skill_context")  # type: SkillContext
         self._config = kwargs
+        if "name" not in self._config:
+            raise ValueError("Missing name of skill component.")
+        self._name = self._config.pop("name")
```

Comment:
the name of a skill component is important since it is the itentifier for a component within the set of components of the same types of a given skill (e.g. the behaviour name is an identifier for all the behaviours of a certain skill).